### ISSUE TITLE
Scratchpad for newm

### ISSUE THREAD

**Issue by iCultureBud** [jbuchermn/newm#70]
One thing I miss a bit is a scratchpad like sway has.

Is something like this planned or wanted?

If not planned, I offer myself to give it a try ;)

**Comment by jbuchermn** [comment]
That's a good idea. Something like this (Conceptually) is actually already there - the launcher panel. This is just a terminal placed off-screen and running `newm-panel-basic launcher`, which can be moved on-screen by a keybinding or a 5-finger gesture.

Possibly this can be repurposed as it is, probably the solution should be to extend / generealize the concept. If you want to take a look, check out the file view.py and take a look for the `View.panel` attribute.

**Comment by iCultureBud** [comment]
Yeah I had the same thought. So I'll take a look at it and create a PR.

**Comment by Pandademic** [comment]
@iCultureBud how's it going working on this?
